src/components/AdminPage/AdminPanel/AdminPanel.js

```diff
@@ -1,5 +1,92 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import styles from "./AdminPanel.module.css";
+import { auth } from "../../../firebase";
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { Button, Form, FormGroup, FormControl } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Title from "../../Title/Title";
+
 const AdminPanel = () => {
-  return <div>AdminPanel</div>;
-};
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [, setCookie] = useCookies(["isSignedIn"]);
+  const navigate = useNavigate();
+
+  const navigation = () => {
+    navigate("/admin/home");
+  };
 
+  const signInUser = () => {
+    signInWithEmailAndPassword(auth, email, password)
+      .then((res) => {
+        setCookie("isSignedIn", true, { path: "/" });
+        navigation();
+      })
+      .catch((err) => {
+        alert("Invalid e-mail or password!");
```

Comment:
When it comes to handling errors, we can leave this alert for now, but here you will have to give another component.